unit_tests/tests/core/slot_toggled_field.rs

```diff
@@ -0,0 +1,48 @@
+#[cfg(test)]
```

Comment:
you can throw all these into the slot_toggled_field file at the bottom

Reply:
Can do